packages/storybook8/stories/Components/TypingIndicator/Docs.mdx

```diff
@@ -0,0 +1,33 @@
+import { Canvas, Meta, Source } from '@storybook/blocks';
+import TypingIndicatorExampleText from '!!raw-loader!./snippets/TypingIndicator.snippet.tsx';
+import CustomizeStyleExampleText from '!!raw-loader!./snippets/CustomizeStyle.snippet.tsx';
+import CustomizeUserRenderingExampleText from '!!raw-loader!./snippets/CustomizeUserRendering.snippet.tsx';
+import * as TypingIndicatorStories from './index.stories';
+
+<Meta of={TypingIndicatorStories} /># Typing Indicator Component to notify the local user when one or more participants
```

Comment:
Should `# Typing Indicator Component to notify the local user when one or more participants
Comment` be wrapped in another new line

Reply:
Odd looks like a linter forced this edit. Fixed the formatting